packages/simcore-sdk/tests/unit/test_node_ports_v2_port.py

```diff
@@ -546,6 +580,7 @@ class FakeNodePorts:
         user_id: int
         project_id: str
         node_uuid: str
+        follow_symlinks: bool = True
```

Comment:
why True?

Reply:
By default we allow to follow symlinks. To disable this, it must put it `False`.  If a symlink is used in this and symlinks are disabled an error will be raised.

Reply:
The node ports is created dynamically. I would expect to setup such a setting (or its default there)
I do not understand why the test fixture has a default value. It kinds of defeat the purpose no?

Reply:
You can set it up. When you instantiate nodeports, the default value is `True` in that case as well.

Reply:
yes but why having a default in a test fixture? in a test you have to define things completely. we do not want things that are defined by default there, they generate confusion when we do not understand how it gets set later on.
I would definitely vote for removing any default in a test argument or else fix the other test cases that use that fixture.

Reply:
Removing this as agreed, also informed @pcrespov about it.